### PR TITLE
Add mendix githubactions

### PR DESCRIPTION
**Pull request by valdore86**

### CODE REVIEW

**Comment by valdore86** [comment]
@dennis-sig @MichielCuijpers

**Review by MichielCuijpers — CHANGES_REQUESTED** [review]
- add SIGRID_CI_PUBLISH: 'publish'
- remove the echo of the token

File: docs/mendix-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Note the name of the branch, which is `main` in the example but might be different for your repository. In general, most older projects will use `master` as their main branch, while more recent projects will use `main`. 
+
+Next, we create a separate workflow for the pull request integration. This will compare the contents of the pull request against the main/master branch from the previous step. In your GitHub repository, create a file `.github/workflows/sigrid-pullrequest.yml` and give it the following contents:
+
+```
+name: sigrid-pullrequest
+on: [pull_request]
+jobs:
+  container:
```
Comment (MichielCuijpers) [inline docs/mendix-github-actions.md:89]: you can rename 'container' into SigridCI-for-QSM

Comment (valdore86) [reply docs/mendix-github-actions.md:89]: Renamed

**Comment by valdore86** [comment]
> * add SIGRID_CI_PUBLISH: 'publish'
> * remove the echo of the token

I've removed the echo from the publish and pull-request.
I'm not sure. what you mean with `add SIGRID_CI_PUBLISH: 'publish'`

**Review by MichielCuijpers — APPROVED** [review]
LGTM

**Pull request merged**